deploy/kubernetes/examples/demo-storageclass.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: demo-storageclass
 provisioner: block.csi.ibm.com
 parameters:
-  SpaceEfficiency: deduplicated   # Optional.
+  SpaceEfficiency: demo-space-efficiency  # Optional.
```

Comment:
for non-user-defined values, our current examples convention is to use one of the possible values

Reply:
but there is no value that you can put there that is actually cross storage.

Reply:
true, but still, we use predefined values that work at least for svc.
this parameter is not even relevant for xiv, so it's not like `demo-space-efficiency` is more "cross storage"

Reply:
`thin` will be the most "cross storage"y

Reply:
I changed it to thin